src/main/java/frc/robot/subsystems/ControlPanelSubsystem.java

```diff
@@ -0,0 +1,79 @@
+package frc.robot.subsystems;
+
+import java.util.List;
+
+import com.revrobotics.ColorSensorV3;
+import com.revrobotics.ColorMatch;
+
+import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.I2C.Port;
+import edu.wpi.first.wpilibj.util.Color;
+
+public class ControlPanelSubsystem extends SubsystemBase {
+
+    public VictorSPX cpMotor;
+    public DoubleSolenoid cpSolenoid;
+    public ColorSensorV3 colorSensor;
+    public DriverStation ds;
+    public ColorMatch colorMatcher;
```

Comment:
Generally it's good to make this private and use getters and setters so that if you need to change the behavior of accessing them you can do it all in one place

Reply:
Ahh... gotta love Java